obspy/imaging/waveform.py

```diff
@@ -1229,6 +1229,9 @@ def __sect_scale_traces(self):
     def __sect_init_time(self):
         """
         Define the time vector for each trace
+        If you uhave supplied negative distances will compute the vred
```

Comment:
whoops ,  my bad





On 6/04/2016, at 12:21 pm, Elliott Sales de Andrade <notifications@github.com> wrote:

> In obspy/imaging/waveform.py:
> 
> > @@ -1229,6 +1229,9 @@ def __sect_scale_traces(self):
> >      def __sect_init_time(self):
> >          """
> >          Define the time vector for each trace
> > +        If you uhave supplied negative distances will compute the vred
> uhave -> have
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>